src/components/charts/other.js

```diff
@@ -350,6 +366,39 @@ export default class Other extends Component {
     this.setState({ configurations });
   }
 
+  onWordcloudInputChange(event) {
+    const { configurations } = this.state;
+    const newValue = event.target.value === ''
+      ? 'Enter here your text to plot its words set on the chart...'
```

Comment:
The chart does not work with an empty data...

Reply:
In my opinion, if you remove all text on focus event, the usage of this textarea will be more comfortable.
Especially, when user copy and paste text from another place.

Reply:
I agree! Good idea! Thank you!